packages/core/util/util.js

```diff
@@ -188,6 +188,15 @@ const mergeClassNameMaps = (baseMap, additiveMap, allowedClassNames) => {
 				css[key] = baseMap[key] + ' ' + additiveMap[key];
 			}
 		});
+
+		if (process.env.NODE_ENV === 'test') {
```

Comment:
Wondering if we should add a special enact-specific guard here as well since it is possible (though unlikely) that a third party might be using this without the same plugins and this could fail their tests since it would change behavior.

Reply:
We should be good. with just `process.env.NODE_ENV === 'test'`. Why it is generic enough to match other testing tools, the csss identity name spec matches our primary compatibility targetof create-react-app, both for modular and static css. Any custom enact env var would potentially break CRA from working on our tests as expects, so I'm fine with this current propsal as-is personally.